.github/workflows/deploy-dev-pkgs.yml

```diff
@@ -0,0 +1,82 @@
+name: Deploy "conda" conda packages
+
+on:
+  workflow_run:
+    workflow:
+      - CI tests
+    branches:
+      - master
+    types:
+      - completed
+
+jobs:
+  # Check for required configuration/secrets needed to build & upload.  Done
+  # this way because secrets cannot be used directly in GitHub Action
+  # conditionals; see https://github.com/actions/runner/issues/520 for details.
+  check-config:
+    runs-on: ubuntu-18.04
```

Comment:
Can we use ubuntu-20.04 here?

Reply:
Yup, can do. The recipe doesn't depend on anything from the underlying OS, so there's no reason not to.